extendr-api/src/robj_ndarray.rs

```diff
@@ -127,3 +159,61 @@ fn test_from_robj() {
         // assert_eq!(mx[[3, 1]], 8);
     }
 }
+
+#[test]
+fn test_to_robj() {
+    test! {
+        assert_eq!(
+            &Robj::try_from(&array![1., 2., 3.])?,
+            &R!("array(c(1, 2, 3))")?
+        );
+
+        // We give both R and Rust the same 1d vector and tell them both to read it as a matrix
+        // in C order. Therefore these arrays should be the same.
+        assert_eq!(
+            &Robj::try_from(&Array::from_shape_vec((2, 3), vec![1., 2., 3., 4., 5., 6.]).unwrap())?,
+            &R!("matrix(c(1, 2, 3, 4, 5, 6), nrow=2, byrow=T)")?
```

Comment:
One minor nitpick: Please write out `TRUE` and `FALSE`. `T` and `F` are bad style and it's best to get used to never use them. (They are just variables that have been assigned `TRUE` and `FALSE`, respectively.)